examples/atomic/Kconfig

```diff
@@ -0,0 +1,29 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_ATOMIC
+	tristate "\"Test atomic\" example"
```

Comment:
Why test atomic is part of "examples" and not "testing"?

Reply:
The code is a simple demo to check the atomic work as expect, so it's fine to put either examples or testing.

Reply:
done